rpcs3/Emu/RSX/GL/GLHelpers.h

```diff
@@ -1890,6 +1890,11 @@ namespace gl
 		{
 			return{ component_swizzle[3], component_swizzle[0], component_swizzle[1], component_swizzle[2] };
 		}
+
+		u32 encoded_component_map() const
+		{
+			return (5 << 0 | 2 << 3 | 3 << 6 | 4 << 9);
```

Comment:
magic constants?

Reply:
This method is unused, really should just return 0 here. I could also add a macro for it instead and call it DEFAULT_COMPONENT_MAP.